src/rules.scala

```diff
@@ -430,6 +434,7 @@ object Plugin extends sbt.Plugin {
     libraryProjects          := localProjects.value ++ apklibs.value ++ aars.value,
     libraryProject          <<= properties { p =>
       Option(p.getProperty("android.library")) exists { _.equals("true") } },
+    checkAars               <<= checkAarsTaskDef,
```

Comment:
hook checkAars into either `aars` or `collectResources`, whichever makes sense for its dependencies